pydis_site/templates/home/index.html

```diff
@@ -7,65 +7,88 @@
-          The hottest Python community on the web
-        </h1>
-      </div>
-    </div>
-    <div class="hero-foot">
-      {% include "base/navbar.html" with active_item=False dropdowns=True icon_weight="fas" use_logo=False %}
-    </div>
-  </section>
+  {% include "base/navbar.html" %}
+
   <section class="section">
+
+    <!-- Who are we? -->
     <div class="container is-spaced">
+      <h1 class="is-size-1">Who are we?</h1>
+      <br>
       <div class="columns">
-        <div class="column is-two-thirds">
+        <div class="column is-half">
           <p>
-            We're a large, friendly community focused around the Python programming
-            language, open to those who wish to learn the language or improve their
-            skills, as well as those looking to help others.
+            We're a large community focused around the Python programming language.
+            We believe anyone can learn programming, and are very dedicated to helping
+            novice developers take their first steps into the world of code. We also
+            attract a lot of expert developers who are seeking friendships, collaborators,
+            or looking to hone their craft by teaching and getting involved in the community.
 
-            <br /><br />
+            <br/><br/>
 
-            We organise regular community events and have a dedicated staff of
-            talented Python developers available to assist around the clock.
+            We organise regular community events like code jams, open source hackathons,
+            seasonal events and community challenges. Through our sponsorships and with
+            help from donations, we are even able to award prizes to the winners of our events.
 
-            <br /><br />
+            <br/><br/>
 
-            Whether you're looking to learn the language or working on a complex project,
-            we've got someone who can help you if you get stuck.
+            You can find help with most Python-related problems in one of our help channels.
+            Our staff of nearly 50 dedicated expert Helpers are available around the clock
+            in every timezone. Whether you're looking to learn the language or working on a
+            complex project, we've got someone who can help you if you get stuck.
           </p>
         </div>
-        <div class="column is-one-third is-first has-text-centered">
-          <h1 class="title">Join the community</h1>
 
-          {# TODO: Fix URL #}
-          <a href="https://pythondiscord.com/invite">
-            <img class="discord-banner" src="https://discordapp.com/api/guilds/267624335836053506/embed.png?style=banner3" alt="Discord banner" />
-          </a>
+        <!-- Intro video -->
+        <div class="column is-half video-container">
+          <iframe src="https://www.youtube.com/embed/DIBXg8Qh7bA" frameborder="0"
+                  allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
```

Comment:
this says autoplay but it didnt start playing for me, is that intentional? autoplay is a horrible thing

Reply:
I'm not sure, this embed is just generated by YouTube, it never autoplays. I removed it and it doesn't appear to change anything. Maybe it just allows an autoplay setting to be toggled somewhere else?